src/data_retrieval/cds/era5.py

```diff
@@ -15,6 +15,8 @@ def get_era5_cds_metadata(
         "month": [f"{month:02d}" for month in months],
         "time": "00:00",
         "format": file_type,
+        # 'area':[32.77,2.09,48.28,15.11],
```

Comment:
these commented lines are a "dead code" and it is a bad practice to leave it behind. Please remove

Reply:
should I add a comment like the one in line 27 on ecmwf.py? explaining that area parameter has been removed to target the globe? or is not necessary ?

Reply:
@ediakatos I am afraid your code is broken

when running your branch with following default values
```bash
poetry run python src/data_retrieval cds ecmwf
```

the script successfully sends the request to the ECMWF queue, but with file path that is total nonsense.

in my case
```
ecmwf-monthly-seasonalforecast-1981-2023.grib/ecmwf_global_forecast_1981_lt1./home/{my-user-name}/Downloads/ecmwf_global_forecast
```

As expected, the process will crash with error as soon as the file is ready for download.

Please fix the file path formatting and make sure you run the tool all the way after you make the changes to ensure your edits are really working.

Reply:
there was a logic mistake in me giving the paths in both main and ecmwf scripts , I changed it and now Im testing all before push